README.md

```diff
@@ -1,4 +1,47 @@
 metro
 =====
 
-Module for sending metric to configurable pipelines
+Node module for sending metrics to graphite time series DB
+
+How to use metro in your repo:
+==============================
+
+1. `npm install metro -s`
```

Comment:
I'm afraid this will something else, a javascript bundle for react (https://www.npmjs.com/package/metro).

We haven't published this as an npm package on the npm registry, installation is with the github repo